__tests__/Unit/Components/Tasks/SuggestionList.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import SuggestionList from '@/components/tasks/SuggestionBox/SuggestionList';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+
+const gitInfo: GithubInfo[] = [
+    {
+        github_id: 'fakhruddinkw',
+        profileImageUrl: placeholderImageURL,
+    },
+    {
+        github_id: 'iamitprakash',
+        profileImageUrl: placeholderImageURL,
+    },
+];
+
```

Comment:
please add this into separate file as fixtures and import here and use it here

Reply:
+1

Reply:
We have no fixtures folder anywhere in the repo @bharati-21 @Pratiyushkumar